x-pack/test/functional_endpoint/page_objects/endpoint_alerts_page.ts

```diff
@@ -23,5 +47,58 @@ export function EndpointAlertsPageProvider({ getService }: FtrProviderContext) {
       await testSubjects.setValue('superDatePickerAbsoluteDateInput', timestamp);
       await this.submitSearchBarFilter();
     },
+    /**
+     * Finds a table and returns the data in a nested array with row 0 is the headers if they exist.
+     * It uses euiTableCellContent to avoid poluting the array data with the euiTableRowCell__mobileHeader data.
+     * @param dataTestSubj
+     * @param element
+     * @returns Promise<string[][]>
+     */
+    async getEndpointAlertResolverTableData(dataTestSubj: string, element: string) {
```

Comment:
this is great that its in a function, have we had any info from Devs on how stable we think the resolver data layout will be?  just wondering about maintenance here.